src/Core/Resources/SharedResources.en.resx

```diff
@@ -585,6 +585,15 @@
     <value>Require users to save vault items to an organization by removing the personal ownership option.</value>
   </data>
   <data name="PersonalOwnershipExemption" xml:space="preserve">
+    <value>Organization users that can manage the organization's policies are exempt from this policy's enforcement.</value>
+  </data>
+  <data name="DisableSend" xml:space="preserve">
+    <value>Disable Send</value>
```

Comment:
Be sure to add an explanation that the word "Send" should not be translated.

Reply:
See my comment on basically every string that has Send in it from the web vault. We can use that in all apps